app/src/main/java/io/intrepid/contest/screens/contestcreation/namecontest/NameContestFragment.java

```diff
@@ -40,6 +40,11 @@ public final void onTextChanged(CharSequence newName) {
     }
 
     @Override
+    public void onFocus() {
+        onTextChanged(contestNameField.getText());
```

Comment:
sure this method calls the presenter's onTextChanged method but let's make this more clear and explicit how it was before.